app/models/user.rb

```diff
@@ -17,6 +17,10 @@ class User < ApplicationRecord
   # -- Callbacks ------------------------------------------------------------
   # -- Validations ----------------------------------------------------------
   # -- Scopes ---------------------------------------------------------------
+  # Preferred notification frequencies
+  scope :digests_daily, -> { where('preferences LIKE "%digest_frequency: daily%"') }
```

Comment:
These scopes could be placed on the `digest_mailer` itself?

Reply:
Makes sense since I think we'll only be using this in the context of digests.